### PR TITLE
Updated Apache NetBeans IDE

### LINKED ISSUE #3714: Netbeans icon lines too thin
As pointed out by @beatrizsmerino in https://github.com/simple-icons/simple-icons/pull/1885#issuecomment-706364905 the current version of the Netbeans icon is not very legible at small sizes (as shown below). For anyone addressing this, consider #2887 as well 🙂 

preview
image

### PR DESCRIPTION
**Pull request by adamrusted**
apachenetbeanside (3)

**Issue:** Contributes toward #2887
Fixes #3714 
**Alexa rank:** N/A

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Path generated in Adobe Illustrator, using [source](https://apache.org/logos/#netbeans) SVG for reference.
Ran through SVGO CLI and SVGlint.

### CODE REVIEW

**Review by mondeja — CHANGES_REQUESTED** [review]
I'm seeing strange imperfections in the corners. Seems that these also exists in previous version currently available in simple-icons. Could you rebuild from source @adamrusted?

Looks like the imperfections are coming when I combine the individual paths for the lines into a combined object. Will try a different way now.

**Comment by adamrusted** [comment]
Whoops! I edited your comment instead of quoting it....

**Comment by adamrusted** [comment]
@mondeja - give this new path a go. It's not the neatest, but should give us correct geometry.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -478,7 +478,7 @@
         {
             "title": "Apache NetBeans IDE",
             "hex": "1B6AC6",
-            "source": "https://netbeans.apache.org/images/"
+            "source": "https://apache.org/logos"
```
Comment (mondeja) [inline _data/simple-icons.json:481]: Could you add here a foward slash at the end to maintain the consistency with other sources?

```suggestion
            "source": "https://apache.org/logos/"
```

**Review by mondeja — CHANGES_REQUESTED** [review]
The geometry looks spot on now, however I'm seeing a lot of weird coordinates in the intersections. Seems that there are inneficient knots in these places... Could you check that?

image

**Comment by adamrusted** [comment]
@mondeja - had to export with a precision of 4 to get the correct dimensions.

**Review by mondeja — CHANGES_REQUESTED** [review]
Wow, perfect! :+1:
Just one small change. I have been able to remove these 6 coordinates without altering the result, could you remove them?

image

**Review by mondeja — APPROVED** [review]
Perfect :100:
Thank you @adamrusted :upside_down_face:

**Pull request merged**